e2e/cypress/integration/plugins/systemconsole/drawPlugin_spec.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Please move/rename this file as `e2e/cypress/integration/system_console/plugin_upload_spec.js`

Reply:
Renamed spec file